src/Serilog.Settings.AppSettings/Serilog.Settings.AppSettings.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>XML configuration (System.Configuration &lt;appSettings&gt;) support for Serilog.</Description>
+    <VersionPrefix>2.1.3</VersionPrefix>
+    <Authors>Serilog Contributors</Authors>
+    <TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
+    <AssemblyName>Serilog.Settings.AppSettings</AssemblyName>
+    <AssemblyOriginatorKeyFile>../../assets/Serilog.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+    <PackageId>Serilog.Settings.AppSettings</PackageId>
+    <PackageTags>serilog;xml</PackageTags>
+    <PackageIconUrl>https://serilog.net/images/serilog-configuration-nuget.png</PackageIconUrl>
+    <PackageProjectUrl>https://github.com/serilog/serilog-settings-appsettings</PackageProjectUrl>
+    <PackageLicenseUrl>https://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Serilog" Version="2.3.0" />
```

Comment:
I think you could go ahead and reference a _newer_ version of serilog. I would say v2.6.0 has been out for long enough, so it's safe to reference it.